src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC } from 'react'
+import { Typography, Grid, Paper, TextField } from '@material-ui/core'
+
+import BackgroundImage from 'assets/images/background.jpg'
+import { useStyles } from './styles'
+
```

Comment:
It is a good practice to group imports like this:

```ts
// 3rd party imports
import React, { FC } from 'react'
import { Typography, Grid, Paper, TextField } from '@material-ui/core'

// absolute path imports (in most cases there are a lot absolute path imports so they should be also grouped together by their logic)
import BackgroundImage from 'assets/images/background.jpg'

// relative path imports
import { useStyles } from './styles'
```
In most of the cases there are lots of imports in react component files - so grouping them this way helps to keep good readability.